sampler_test.go

```diff
@@ -13,6 +13,42 @@ import (
 	"github.com/uber/jaeger-client-go/utils"
 )
 
+func TestSamplerTags(t *testing.T) {
+	prob, err := NewProbabilisticSampler(0.1)
+	require.NoError(t, err)
+	rate, err := NewRateLimitingSampler(0.1)
+	require.NoError(t, err)
+	remote := &RemotelyControlledSampler{
+		sampler: NewConstSampler(true),
+	}
+	tests := []struct {
+		sampler  Sampler
+		typeTag  string
+		paramTag interface{}
+	}{
+		{NewConstSampler(true), "const", true},
+		{NewConstSampler(false), "const", false},
+		{prob, "probabilistic", 0.1},
```

Comment:
strings are intentional because these values are a part of the public contract (e.g. with XYS).